cpp/ql/src/semmle/code/cpp/ir/implementation/raw/internal/InstructionTag.qll

```diff
@@ -2,7 +2,10 @@ private import cpp
 
 newtype TInstructionTag =
   OnlyInstructionTag() or // Single instruction (not including implicit Load)
+  InitializeThisAddressTag() or
```

Comment:
You could probably just reuse the tags we already use when generating other `InitializeParameter` sequences. Not a big deal either way, though.